### PR TITLE
Add Builder 3.2.3

### PR DESCRIPTION
**Pull request by ehelms**
Note that this includes the scratch build re-work to test adding a new package until thats merged

### CODE REVIEW

**Comment by ehelms** [comment]
[test]

**Comment by ehelms** [comment]
[test copr]

**Comment by ehelms** [comment]
[test copr]

File: tasks/build_wait_and_download.yml [hunk 1]
```diff
@@ -9,9 +9,9 @@
   command: "copr-cli watch-build {{ build_tasks | join(' ') }}"
   ignore_errors: yes
   register: build_status
-  when: (target_release | succeeded) and (normal_execution_check is not defined)
+  when: target_release | succeeded)
```
Comment (evgeni) [inline tasks/build_wait_and_download.yml]: https://xkcd.com/859/

**Comment by ehelms** [comment]
I was asking myself just that this morning but they seem to have different structures that do not coincide easily.

**Pull request merged**